addons/stock/tests/test_quant.py

```diff
@@ -58,6 +58,11 @@ def setUpClass(cls):
             'usage': 'internal',
             'location_id': cls.stock_location.id,
         })
+        cls.stock_subloc3 = cls.env['stock.location'].create({
```

Comment:
I would create it before subloc2 to ensure that order is done by name and not id